### PR TITLE
added columns to existing models as well as new models

### PR DESCRIPTION
**Pull request by bahadasx**
Issue #36 - Added new columns to existing data models to account for changes in schema.  Also, added the following new models:  Trip, TripEvent, Region, EventType, Agency, Office, CountryRegion.  The following models are still contained in the models.py file, but should not be used going forward (will be removed in a future pull request): EventLocationPrincipalTravel, Location, and Travel.

### CODE REVIEW

**Comment by arowla** [comment]
We're not actually removing Event, right? N:N to Trip through TripEvent?

**Comment by bahadasx** [comment]
Sorry, no we are not removing event.  I updated my original pull request comment to reflect this.

**Comment by arowla** [comment]
You need to commit the migration file for this.

**Comment by arowla** [comment]
Please make sure PR names reflect the PR topic.

File: travel/admin.py [hunk 1]
```diff
@@ -2,10 +2,17 @@
...
 # Register your models here.
 
-from .models import Event, Location, Principal, Travel, EventLocationPrincipalTravel
+from .models import Event, Location, Principal, Travel, EventLocationPrincipalTravel, Trip, TripEvent, Region, EventType, Agency, Office, CountryRegion
 
 admin.site.register(Event)
 admin.site.register(Location)
 admin.site.register(Principal)
 admin.site.register(Travel)
 admin.site.register(EventLocationPrincipalTravel)
```
Comment (arowla) [inline travel/admin.py]: Is there a reason to leave the admin in for these, or should we disable from the Admin as a half-measure to show that those models are deprecated?

**Comment by arowla** [comment]
This is a lot of migrations to look over at once, and I'm afraid I will miss something about the DB changes. What would you think, @bahadasx, about attempting a [migration squash](https://docs.djangoproject.com/en/1.9/topics/migrations/#squashing-migrations)?

**Comment by bahadasx** [comment]
Sure, I can do that.

**Comment by arowla** [comment]
Migrations 002 and 003 each have two versions-- only one of each of those is included in the squash, apparently. Can you make sure the right ones got squashed (see the `replaces = [ ... ]` line in the squash), and `git rm` the extras?

**Pull request merged**